crm_claim/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Manage Customer Claims.
+=======================
+
+This application allows you to track your customers/vendors claims and
+grievances.
+
+It is fully integrated with the email gateway so that you can create
+automatically new claims based on incoming emails.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to new menu **CRM > Configuration > Claim > Categories** and create as
+  many categories as you need.
+* Go to new menú **CRM > After Sale > Services > Stages** and create stages for
+  claims.
```

Comment:
Sale > Configuration > Claim > Stages

Reply:
This menu exists if you have installed sales module. If not, correct menu is CRM > Configuration > Claim > Stages

This module only depends on sales_team.